pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/rules/ItemsRule.scala

```diff
@@ -79,51 +105,46 @@ object ItemsRule extends FieldMergeRule with MergerLogging {
              sources: NonEmptyList[TransformedBaseWork]): FieldData = {
       val sierraItems = target.data.items
       val metsItems = sources.toList.flatMap(_.data.items)
-      val metsUrls = metsItems.flatMap(_.locations).collect {
-        case DigitalLocation(url, _, _, _, _, _) => url
-      }
+
       debug(s"Merging METS items from ${describeWorks(sources)}")
       sierraItems match {
         case List(sierraItem) =>
           List(
             sierraItem.copy(
-              locations = sierraItem.locations.filterNot(hasUrl(metsUrls)) ++
-                metsItems.flatMap(_.locations)
+              locations = sierraItem.locations ++ metsItems.flatMap(_.locations)
             )
           )
-        case _ =>
-          sierraItems.filterNot(_.locations.exists(hasUrl(metsUrls))) ++
-            metsItems
+        case multipleItems =>
+          multipleItems ++ metsItems
       }
     }
   }
 
-  private val mergeMiroPhysicalAndDigitalItems = new PartialRule {
-    val isDefinedForTarget: WorkPredicate = WorkPredicates.sierraWork
-    val isDefinedForSource
-      : WorkPredicate = WorkPredicates.miroWork or WorkPredicates.digitalSierra
+  /** We merge the Miro location to the Sierra work with a single item.
+    * If there are multiple items, we don't merge as we wouldn't know
+    * which item to attache the location to.
```

Comment:
Why can't we do the same as with other types (ie, appending the items)?

Reply:
We want it to be a separate work, but the question as to why, I am lost. Ping @jtweed

Reply:
Feel like this might be related to https://github.com/wellcomecollection/catalogue/pull/521#discussion_r409647521

Reply:
Also somehow relevant, I remember I made this after a conversation with @jtweed https://github.com/wellcomecollection/platform/issues/4299